installers/osie/main.go

```diff
@@ -12,6 +13,8 @@ func init() {
 	job.RegisterDistro("alpine", bootScripts["rescue"])
 }
 
+var logger log.Logger
```

Comment:
better to use the `installers.Logger()` where needed in the init function than in a global var.